ci-operator/config/openstack-k8s-operators/neutron-operator/openstack-k8s-operators-neutron-operator-master.yaml

```diff
@@ -88,17 +92,17 @@ tests:
   optional: true
   steps:
     workflow: openstack-k8s-operators-test-build-deploy
-- as: neutron-operator-build-deploy-kuttl
+- as: neutron-operator-build-deploy-tempest
```

Comment:
kuttl tests need to stay - pls look at keystone on how to run both together

Reply:
done